deployments/pulse3d/manifests/overlays/test/kustomization.yaml

```diff
@@ -6,3 +6,27 @@ resources:
 - ../../base
 - curibio-jwt-secret.yaml
 - curibio-jobs-creds.yaml
+
+patchesJson6902:
+- target:
+    kind: Ingress
+    name: pulse3d-ingress
+  patch: |-
+    - op: replace
+      path: /spec/rules/0/host
+      value: "pulse3d.curibio-test.com"
+- target:
+    kind: Deployment
+    name: pulse3d
+  patch: |-
+    - op: add
+      path: /spec/template/spec/containers/0/image
+      value: "077346344852.dkr.ecr.us-east-2.amazonaws.com/pulse3d_api:0.5.0"
```

Comment:
You really only need to add this one since it's the one you removed from pulse3d-dep.yaml. The other values default to the test cluster and that's why they get replaced/added in the prod overlay